lib/ansible/modules/cloud/openstack/os_security_group.py

```diff
@@ -117,7 +117,9 @@ def main():
 
     try:
         cloud = shade.openstack_cloud(**module.params)
-        secgroup = cloud.get_security_group(name)
+        project_id = cloud.current_project_id
```

Comment:
This property was only added to shade in 1.13.0. See os_router for an example of checking the shade version.